### PR TITLE
Dp kidney

### PR DESCRIPTION
**Pull request by daniellepace**
Adding:
1) tensorization and tensormap options for pancreas image segmentation
2) additional options for machine learning setup and median T1 time inference, created to emulate Marcus's setup (although in the end these were not used, they might be useful for other projects?)
3) various bug fixes

### CODE REVIEW

File: ml4h/arguments.py [hunk 1]
```diff
@@ -385,10 +385,14 @@ def parse_args():
...
 
     # Arguments for explorations/infer_stats_from_segmented_regions
-    parser.add_argument('--analyze_ground_truth', default=True, help='Whether or not to filter by images with ground truth segmentations, for comparison')
+    parser.add_argument('--analyze_ground_truth', action='store_true', help='Filter by images with ground truth segmentations, for comparison')
+    parser.add_argument('--no_analyze_ground_truth', dest='analyze_ground_truth', action='store_false', help='Do not filter by images with ground truth segmentations, for comparison')
+    parser.set_defaults(analyze_ground_truth=True)
     parser.add_argument('--structures_to_analyze', nargs='*', default=[], help='Structure names to include in the .tsv files and scatter plots')
     parser.add_argument('--erosion_radius', default=1, type=int, help='Radius of the unit disk structuring element for erosion preprocessing')
     parser.add_argument('--intensity_thresh', type=float, help='Threshold value for preprocessing')
+    parser.add_argument('--intensity_thresh_perc', type=float, help='Threshold percentile for preprocessing, between 0 and 100 inclusive')
```
Comment (lucidtronix) [inline ml4h/arguments.py:394]: perc -> percent, we try to use full words in args

File: ml4h/models/layer_wrappers.py [hunk 2]
```diff
@@ -52,9 +53,14 @@
     # class name -> (dimension -> class)
     'spatial_dropout': {2: SpatialDropout1D, 3: SpatialDropout2D, 4: SpatialDropout3D},
     'dropout': defaultdict(lambda _: Dropout),
+    'l1': L1,
```
Comment (lucidtronix) [inline ml4h/models/layer_wrappers.py:56]: nice!

**Review by lucidtronix — APPROVED** [review]
lgtm!

File: ml4h/arguments.py (hunk 1 above)
Comment (daniellepace) [reply ml4h/arguments.py]: Done! Renamed to percentile

**Pull request merged**